### PR TITLE
Fixed #26037 -- Documented precedence of USE_X_FORWARDED_HOST/PORT settings.

### PR DESCRIPTION
**Pull request by arnaudlimbourg**

### CODE REVIEW

File: docs/ref/settings.txt [hunk 1]
```diff
@@ -2524,6 +2524,8 @@ A boolean that specifies whether to use the X-Forwarded-Host header in
 preference to the Host header. This should only be enabled if a proxy
 which sets this header is in use.
 
+.. note: This takes priority over USE_X_FORWARDED_PORT.
```
Comment (timgraham) [inline docs/ref/settings.txt]: Please use
```
:setting:`USE_X_FORWARDED_PORT`
```
to create a link for the setting. You're missing a second colon for `note::` so the block doesn't actually appear in the rendered version.

**Comment by timgraham** [comment]
I'm not sure this provides a detailed enough explanation of the issues raised in the ticket.

**Comment by arnaudlimbourg** [comment]
@timgraham I tried to improve the explanation and got your fixes in.

Is that going in a good direction?

**Comment by timgraham** [comment]
@benoitbryon, @mattrobenolt, do you think these notes are sufficient?

**Comment by mattrobenolt** [comment]
Seems reasonable. :cookie:

**Comment by timgraham** [comment]
merged in 5cda4677b3df1be971000ef27470d3efc308d3be, thanks!

**Pull request closed without merging**